components/EventItem.tsx

```diff
@@ -1,326 +1,176 @@
-import { StyleSheet, Text, View } from "react-native"
+import React from "react"
+import { Image, StyleSheet, Text, View } from "react-native"
 import { Event } from "@lib/events/Event"
 import IconButton from "./common/IconButton"
 import { Divider, Icon } from "react-native-elements"
+import { Shadow } from "react-native-shadow-2"
+import tinycolor from "tinycolor2"
+import { placemarkToFormattedAddress } from "@lib/location"
 
 interface Props {
   event: Event
 }
 
 const EventItem = ({ event }: Props) => {
-  const [requested, setRequested] = useState(false) // If user has requested to join
-  const [numInvitations, setNumInvitations] = useState(0) // Number of requested invitations
-  const [isMin, setIsMin] = useState(false) // If time limit has <> 1 hour left
-  const [isHours, setIsHours] = useState(false) // If time limit has >= 1 hour left
-  const [isDays, setIsDays] = useState(false) // // If time limit has > 24 hours left
-  const [timeUntil, setTimeUntil] = useState(0) // Time (in either days, hours, or minutes) until event starts
-  const [currentCapacity, setCurrentCapacity] = useState(1) // How many users have joined event
-  const CAPACITY_PERCENTAGE = 0.75
+  const numAttendees = 1
+  const distance = 0.5
+  const shadowColor = "#bdbdbd"
+  const lightEventColor = tinycolor(event.colorHex).lighten(10).toString()
 
-  const setTime = () => {
-    if (event.startTime) {
-      const date = new Date()
-      const diffTime = Math.abs(event.startTime.getTime() - date.getTime())
-      const diffMin = Math.ceil(diffTime / (1000 * 60))
-      const diffHours = Math.floor(diffMin / 60)
-      const diffDays = Math.floor(diffHours / 24)
-
-      if (diffMin < 60) {
-        setIsMin(true)
-        setIsHours(false)
-        setIsDays(false)
-        setTimeUntil(diffMin)
-      } else if (diffHours < 24) {
-        setIsMin(false)
-        setIsHours(true)
-        setIsDays(false)
-
-        if (diffMin % 60 >= 20 && diffMin % 60 <= 40) {
-          setTimeUntil(diffHours + 0.5)
-        } else {
-          setTimeUntil(diffHours)
-        }
-      } else {
-        setIsMin(false)
-        setIsDays(true)
-        setIsHours(false)
-        setTimeUntil(diffDays)
-      }
-    }
+  const onPressMore = () => {
+    return null
   }
 
-  useEffect(() => {
-    setTime()
-  }, [event.startTime])
-
-  const handleRequestToJoin = () => {
-    if (requested) {
-      setRequested(false)
-      setNumInvitations(numInvitations - 1)
-    } else {
-      setRequested(true)
-      setNumInvitations(numInvitations + 1)
-    }
-  }
+  return (
+    <Shadow
+      distance={5}
+      startColor={shadowColor}
+      offset={[0, 3]}
+      style={{ alignSelf: "stretch" }}
+    >
+      <View style={[styles.container]}>
+        {/* Profile Image, Name, More button */}
+        <View style={[styles.topRow, styles.flexRow]}>
+          <Image
+            style={styles.image}
+            source={require("../assets/icon.png")}
```

Comment:
We could create a clone of the profile image and name component with this new styling since this will be used a lot throughout the app. @mhayes853 may want to do refactoring on that component first though 😅